draft-irtf-cfrg-vdaf.md

```diff
@@ -4561,6 +4621,10 @@ We also stress that even if the Idpf is not extractable, Poplar1 guarantees
 that every client can contribute to at most one prefix among the ones being
 evaluated by the helpers.
 
+## Considerations for multiple proofs (`PROOFS`) {#security-multiproof}
+> TODO on how changing `PROOFS` ({{multiproofs}}) impacts security, in particular when we go
```

Comment:
* "TODO"s should be imperative, similarly to how we write commit messages. (E.g., TODO Add guidance for ..."
* Add a reference to the open issue.
* Break lines at 80 characters.